### PR TITLE
Add multixact test reproducing the problem with duplicates caused by incorrect opffset calculation

### PR DESCRIPTION
**Pull request by knizhnik**
## Problem

See #4774

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [ ] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [ ] Do not forget to reformat commit message to not include the above checklist

### CODE REVIEW

**Comment by hlinnaka** [comment]
Thanks! At a quick glance, this seems pretty redundant with the test we have. This does multiple UPDATEs in parallel, from multiple threads, so that might uncover some Postgres bugs.

I don't think we need this, but I'm also not going to say no to having more tests, assuming this test runs fairly quickly. :-)

File: test_runner/regress/test_multixact_conc.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    assert res is not None
+    next_multixact_id = res[0]
+    lsn = res[1]
+
+    # Ensure that we did lock some tuples
+    assert int(next_multixact_id) > int(next_multixact_id_old)
+
+    # Branch at this point
+    env.neon_cli.create_branch("test_multixact_new", "test_multixact", ancestor_start_lsn=lsn)
+    endpoint_new = env.endpoints.create_start("test_multixact_new")
```
Comment (koivunej) [inline test_runner/regress/test_multixact_conc.py:84]: I wonder if we need to create a branch here, would a `env.endpoints.create_start(name, lsn=lsn)` suffice? Or is branching expected to play a role in here?

Compared to just starting a read only endpoint, the new timeline will have gc and comapction working, which has sometimes caused flakyness.

**Comment by knizhnik** [comment]
> Thanks! At a quick glance, this seems pretty redundant with the test we have. This does multiple UPDATEs in parallel, from multiple threads, so that might uncover some Postgres bugs.
> 
> I don't think we need this, but I'm also not going to say no to having more tests, assuming this test runs fairly quickly. :-)

+1
Actually it was one more mishit from my side: trying to reproduce there problem I quite soo get this error (content of restored database doesn't match with the original database), but I incorrectly interpret it - I just thought that multixacts were  not correctly saved.

File: test_runner/regress/test_multixact_conc.py (hunk 1 above)
Comment (knizhnik) [reply test_runner/regress/test_multixact_conc.py:84]: Branch is not needed - I it was just copied from the original version of test_multixact test
